### PR TITLE
Livable code

### PR DESCRIPTION
**Pull request by idabmat**

### CODE REVIEW

File: posts/no-uncle-of-mine.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+> Instead, you counter bad ideas with better ideas.
+
+And as such, not only was the firing a wrong decision, but it would in turn make
+people unconfortable sharing their ideas, which would endanger a company as
+Google that relies on innovation.
+
+While I agree that you should try to discuss things freely, and allow people and
+their ideas to evolve though debate, there is a limit to how far this can go on
+for. In this particular context, it seems like Google did provide
+opportunities both for discussion and learning, as is mentionned in the
```
Comment (craigjbass) [inline posts/no-uncle-of-mine.md]: mentionned

File: posts/no-uncle-of-mine.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+- The manifesto _does_ show [sexism](https://medium.com/@adljksbvkj/heres-your-point-by-point-refutation-of-the-google-memo-b7201d0cca04).
+- And my personal favorite: Sharing such a document to the whole company turned
+  the author into someone people could no longer [work with](https://medium.com/@yonatanzunger/so-about-this-googlers-manifesto-1e3773ed1788).
+
+## Teaching and punishing
+
+Now, amongst the multitude of blog posts, news articles, and tweets that
+followed the release of the memo, one influential person within the software
+industry, let's call him Stan, came out with his take on the matter. Well,
+actually, Stan did not focus on the content of the memo, which he aknowledge was
```
Comment (RyanMacG) [inline posts/no-uncle-of-mine.md]: should be acknowledged

File: posts/no-uncle-of-mine.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+wrong, but instead focused on Google's decision to fire Googlebro. According to
+him :
+
+> You never punish bad ideas.
+> Instead, you counter bad ideas with better ideas.
+
+And as such, not only was the firing a wrong decision, but it would in turn make
+people unconfortable sharing their ideas, which would endanger a company as
```
Comment (craigjbass) [inline posts/no-uncle-of-mine.md]: unconfortable

File: posts/no-uncle-of-mine.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+# No Uncle of Mine
+
+Last month, one topic sparked a lot of debate on the internet in general, and in
+the Tech industry in particular: the so called "Manifesto", writen by James
+Damore, then engineer at Google. While this blog post won't be about the
+manifesto, some events that occured after it came out prompted me to write this.
+But before looking into it, let's go over some of what has beeen said and
+written since last month.
+
+## Googlebro's Manifesto and subsequent firing.
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: I don't think using "Googlebro" is helping your case

File: posts/no-uncle-of-mine.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+In case you've missed it, you can find the original manifesto [here (PDF)](https://assets.documentcloud.org/documents/3914586/Googles-Ideological-Echo-Chamber.pdf).
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

File: posts/no-uncle-of-mine.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+That piece of writing has definetely received more attention than it deserves,
+to the point of even having a lengthy [Wikipedia entry](https://en.wikipedia.org/wiki/Google's_Ideological_Echo_Chamber).
+Since it became public,  many have discussed this in more detail than I'll be
+able to do here, so to keep this short, here are some interesting reading on the
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: should be *is* some interesting reading..

File: posts/no-uncle-of-mine.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+While I agree that you should try to discuss things freely, and allow people and
+their ideas to evolve though debate, there is a limit to how far this can go on
+for. In this particular context, it seems like Google did provide
+opportunities both for discussion and learning, as is mentionned in the
+manifesto itself. If even after this, someone continues pushing on a subject
+with a rhetoric that hurts the company and its employees, I don't see any other
+path other than the dismissal of the offender.
+
+Furthermore, it is important indeed that people feel like they can share their
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: indeed important over important indeed

File: posts/no-uncle-of-mine.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+ideas freely. So this raises the question, what should a company do when one of
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: So this raises the question*:*

File: posts/no-uncle-of-mine.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+old ones, and you would be excused for hoping these would be considered defeated
+in the 21st century.
+
+## The dangers of missing the point
+
+Some things are not okay to say, especially in an industry such as ours where
+white men are over represented. Thus the need for our community, in particular
+its more privileged members, to stand up and denounce points of view such as the
+one expressed by Damore. Failure in doing so will continue to make this community
+unwelcoming for under representended folks.
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: represented

File: posts/no-uncle-of-mine.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+This is why Stan's stance was probably the most toxic thing a prominent white
+men in the community could have said. As such Stan should stop complaining
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: man

Comment (chrisblackburn) [reply posts/no-uncle-of-mine.md]: could say over could have said (I think)

File: posts/no-uncle-of-mine.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+about the quality of the feedback he received, and instead start asking himself
+some questions as to why people sent him mostly aggressive or condescending
+tweets.
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: Starts getting a little preachy here IMHO - Stan SHOULD DO THIS

File: posts/no-uncle-of-mine.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+Part of the issue also is that Stan has a pretty poor track record when it comes
+to talking about women in tech, and one of his following posts sheds some light
+into why that is. While he has recognized and publicly apologized for some
+misplaced humour in the past, he seems very unapologetic about this current
+issue.
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: I'm not sure we should be chastising someone for not apologising for their current viewpoint? Is that what's being suggested?

File: posts/no-uncle-of-mine.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+## Greater than code
+
+When trying to wrap my head around this, I realized something about Stan. One
+of the reasons he is so important in the software industry is because of his
+thoughts on the way we work. One thing that nows strikes me is how much emphasis
+he puts on professional skills. While this emphasis has many values, bore many
+fruits, and has an historical reason to come into existence, I start feeling
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: a historical

File: posts/no-uncle-of-mine.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+like their importance should become second class in the modern software
+industry. More and more, the challenges we face are about people, making teams
+efficient, and delighting users and customers. Those challenges are the ones
+that actually deliver value, but are by nature greater than just code.
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: *and* are by nature..?

File: posts/no-uncle-of-mine.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+The craftsman analogy may have been good, but as with any analogy it reaches a
+point where it starts to become thin, and others may have to be devised. Those
+are definetly interesting times with interesting challenges to be solved as the
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: definitely

File: posts/no-uncle-of-mine.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+thoughts on the way we work. One thing that nows strikes me is how much emphasis
+he puts on professional skills. While this emphasis has many values, bore many
+fruits, and has an historical reason to come into existence, I start feeling
+like their importance should become second class in the modern software
+industry. More and more, the challenges we face are about people, making teams
+efficient, and delighting users and customers. Those challenges are the ones
+that actually deliver value, but are by nature greater than just code.
+
+The craftsman analogy may have been good, but as with any analogy it reaches a
+point where it starts to become thin, and others may have to be devised. Those
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: These

File: posts/no-uncle-of-mine.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+are definetly interesting times with interesting challenges to be solved as the
+software community continues to evolve and to mature. One possibility could be
+to stop focusing on Clean Code, but start thinking about [Livable Code](https://www.youtube.com/watch?v=UDp6eDCi1Vo)
```
Comment (chrisblackburn) [inline posts/no-uncle-of-mine.md]: but *to* start

File: posts/no-uncle-of-mine.md (hunk 13 above)
Comment (craigjbass) [reply posts/no-uncle-of-mine.md]: I prefer the "an" variant of this. Both are acceptance AFAICR

**Comment by chrisblackburn** [comment]
I think an interesting post on a touchy topic. I think it could have slightly more credibility by losing a little of the bias ("Googlebro", for example). 

I think there's something in Stan's suggestion that it's dangerous to take action such as firing people whose ideas are disagreed with. I think you do a good job of refuting why that can be detrimental in cases such as this. I think it's interesting to assume that Stan was making that suggestion without malice intended, so perhaps a little less chastising may actually improve your arguments.

File: posts/no-uncle-of-mine.md (hunk 13 above)
Comment (sowasred2012) [reply posts/no-uncle-of-mine.md]: +1 for "a historical". Unless we're dropping the H and pronouncing it "anistorical".

**Pull request merged**